docs/software_config.md

```diff
@@ -0,0 +1,21 @@
+Software Configuration from New 
+(note that this will download the full repository and the files could be seperated by client and server if desired)
+April 20th, 2022
+
+1)  Create and go into a desired project folder (currently ~/LFIAS) and run:
+  * git clone https://github.com/rudy-patel/imageAnalysisSystem.git .
+2)  Create a virtual environment, activate it, and install all Python packages from the requirements.txt file, this will take some time
+    (note that the activate command will need to be run to restart the venv to run the program, more info can be found here:
+    https://docs.python.org/3/tutorial/venv.html)
+  * python -m venv env
+  * source env/bin/activate
+  * python -m pip install -r requirements.txt
+3)  Place the AWS config and credentials files in the place where Linux expects them:
+  * sudo mkdir ~/.aws
+  * copy the config and credentials files into this directory
+4)  Export the expected environment variables
+  * export FLASK_APP=main.py
+  * export DB_PASSWORD=dolce5719
```

Comment:
Should we be including passwords in plain text like this? I don't have an alternative other than we just give this info separately.

Reply:
This is a good point, and we should definitely address this.